cumulus_library/template_sql/utils.py

```diff
@@ -160,7 +160,6 @@ def _check_schema_if_exists(
         if cursor.fetchone() is None:
             return False
 
-        query = get_column_datatype_query(schema, table, base_col)
         cursor.execute(query)
```

Comment:
nit: you could just save the result of the previous query rather than re-querying now, yeah?

Reply:
i think this is a debugging cursor state artifact

Reply:
With the second `execute()` call, the second `fetchone()` is getting the _second_ line of results. I assume that's not what you want? You might want to save the results of the first `fetchone()` and re-use it.

Reply:
i think this was actually a mistake to have removed the second query in the first place - after digging in the tests for this file are still using pyathena mocks and i think that masked this issue. Should have thought about this one a bit more when i saw it the first time around - this was more of a rabbit hole picking it up fresh in the morning than i'd thought.